### PR TITLE
Feature/304 polkaassembely discussion list

### LINKED ISSUE #304: Feature: List of discussions on Polkassembly
> As user I'd like to see the different discussions occurring on Polkadot within the app.

For both iOS and Android users

### ⚠️ what do we need?

- [x] UI/UX : Where is the entry point ? A teaser on the dashboard or just in the drawer ?  
- [ ] Should we have this feature it in Litentry Testnet or only in the Kusama and Polkadot network ? (ask @geastwood )
- [x] Add filters to the discussion list: What are the filters

### PR DESCRIPTION
**Pull request by ahoseinian**
fixes #304 
- Add polkaassembely discussions + sort, filter and pagination

https://user-images.githubusercontent.com/1168983/130826916-5f33dcb7-54d4-4456-b390-7b6b6368ead6.mp4

### CODE REVIEW

File: src/presentational/Label.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <Text category="label" appearance="alternative" style={styles.text}>
+        {text}
+      </Text>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: 'grey',
```
Comment (silva-fj) [inline src/presentational/Label.tsx:18]: How does it look with the dark theme?

File: src/presentational/Label.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    paddingHorizontal: 5,
+    paddingBottom: 3,
+    paddingTop: 2,
+    borderRadius: 5,
+  },
+  text: {},
```
Comment (silva-fj) [inline src/presentational/Label.tsx]: to remove?

File: src/screen/Polkadot/PolkadotDiscussions.tsx [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  },
+  footer: {
+    paddingTop: standardPadding * 2,
+    backgroundColor: '#fff',
+    marginBottom: standardPadding * 2,
+  },
+});
+
+// format date to days ago
+function formatDate(date: string) {
```
Comment (silva-fj) [inline src/screen/Polkadot/PolkadotDiscussions.tsx]: Let's put this in a separe file. like `src/utils/date.ts`, later we can move the other general utils there.

This function can just be
```
function fromNow(date: string) {
   return moment(date).fromNow();
}
```

File: src/screen/Polkadot/PolkadotDiscussions.tsx [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+import AddressInlineTeaser from 'layout/AddressInlineTeaser';
+import moment from 'moment';
+import {Label} from 'presentational/Label';
+import {flatten} from 'lodash';
+
+export function PolkadotDiscussions() {
+  const [orderBy, setOrderBy] = React.useState<OrderByType>('lastCommented');
+  const [topicId, setTopicId] = React.useState<number>();
+  const [sortVisible, setSortVisible] = React.useState(false);
+  const [filterVisible, setFilterVisible] = React.useState(false);
```
Comment (silva-fj) [inline src/screen/Polkadot/PolkadotDiscussions.tsx]: How about `filterMenuVisible` and `sortMenuVisible` ?

File: src/presentational/Label.tsx (hunk 1 above)
Comment (ahoseinian) [reply src/presentational/Label.tsx:18]: didn't look great, I changed it to this.


ea6b47e

File: src/presentational/Label.tsx (hunk 2 above)
Comment (ahoseinian) [reply src/presentational/Label.tsx]: done ✅

File: src/screen/Polkadot/PolkadotDiscussions.tsx (hunk 3 above)
Comment (ahoseinian) [reply src/screen/Polkadot/PolkadotDiscussions.tsx]: I just removed it in favor of using moment(date).fromNow() directly. thanks!

File: src/screen/Polkadot/PolkadotDiscussions.tsx (hunk 4 above)
Comment (ahoseinian) [reply src/screen/Polkadot/PolkadotDiscussions.tsx]: f4b0177 
done ✅

**Review by silva-fj — APPROVED** [review]

**Pull request merged**